doc/Tutorials.xml

```diff
@@ -313,6 +313,117 @@ AutoDoc( rec( gapdoc := rec( gap_root_relative_path := true ) ) );
+</Heading>
+
+Here is a checklist for authors of a package &PackageName;, 
+<E>which already has a &GAPDoc; based manual</E>, who wish to use &AutoDoc; 
+to build the manual from now on. 
+We assume that the manual is currently built by reading a file 
+<File>makedoc.g</File> and that the main <File>.xml</File> file 
+is named <File>manual.xml</File>. 
+<P/> 
+The files <File>PackageInfo.g</File>, <File>makedoc.g</File>, 
+<File>doc/title.xml</File> and <File>doc/PackageName.xml</File> 
+(if it exists) will be altered by this procedure, 
+so it may be wise to keep backup copies. 
+<P/> 
+You should have copies of the &AutoDoc; files <File>PackageInfo.g</File> 
+and <File>makedoc.g</File> to hand when reading these instructions. 
+<P/> 
+<List>
+<Item> 
+Copy <File>AutoDoc/makedoc.g</File> to <File>PackageName/makedoc.g</File>. 
+</Item>
+<Item>
+Edit <File>PackageName/makedoc.g</File> as follows. 
+  <List>
+  <Item> 
+  Change the header comment to match other files in your package. 
+  </Item> 
+  <Item> 
+  After <Code>LoadPackage("AutoDoc");</Code> 
+  add <Code>LoadPackage("PackageName");</Code>. 
+  </Item> 
+  <Item> 
+  In the <Code>AutoDoc</Code> record delete <Code>AutoDoc := true;</Code>. 
+  </Item>
+  <Item> 
+  <Index Key="Scaffold record in makedoc.g"></Index> 
+  In the <Code>scaffold</Code> record change the <Code>includes</Code> list 
+  to be the list of your <Code>.xml</Code> files that are contained in  
+  <File>manual.xml</File>. 
+  </Item> 
+  <Item>
+  <Index Key="Bibliography field in makedoc.g"></Index> 
+  If you do not have a bibliography you may delete the 
+  <Code>bib := "bib.xml",</Code> field in the scaffold. 
+  Otherwise, edit the file name if you have a different file. 
+  If you only have a <Code>.bib</Code> file  
+  (<Code>manual.bib</Code> or <Code>bib.xml.bib</Code> say) 
+  you should rename this file <File>PackageName.bib</File>. 
+  </Item>
+  <Item> 
+  In the <Code>gapdoc_latex_options</Code> record enter any 
+  &LaTeX; <Code>newcommands</Code> previously in <File>manual.xml</File>. 
+  (If there are none you may safely delete this record.) 
+  To illustrate this option the &AutoDoc; file <File>makedoc.g</File> 
+  defines the command <Code>\bbZ</Code> which may be used to produce 
+  the &LaTeX; formula <M>f : \bbZ^2 \to \bbZ</M>. 
+  </Item>
+  <Item> 
+  <Index Key="Entities record in makedoc.g"></Index> 
+  In the <Code>entities</Code> record enter any entities 
+  previously stored in <File>manual.xml</File>. 
+  (Again, if you have none, you may safely delete this record.) 
+  </Item> 
+  </List>
+</Item> 
+<Item> 
+Now edit <File>PackageName/PackageInfo.g</File> as follows. 
+  <List>
+  <Item> 
+  Delete any <Code>PKGVERSIONDATA</Code> chunk that may be there. 
+  The main reason for converting your manual to use &AutoDoc; is to stop 
+  using entities such as <Code>PACKAGENAMEVERSION</Code>. 
+  </Item>
+  <Item>
+  Add <Code>[ "AutoDoc", ">= &VERSION;" ]</Code>  
```

Comment:
Do we really want to suggest that minimal version?

Or, for that matter, do we really want people to add AutoDoc to their `SuggestedOtherPackages`? I see no advantage in doing that.

Reply:
My mistake - removed the instruction to put AutoDoc in SuggestedOtherPackasges . . .